addons/web_editor/static/src/js/wysiwyg/fonts.js

```diff
@@ -36,7 +36,7 @@ return {
                 // stylesheets.
                 rules = sheets[i].rules || sheets[i].cssRules;
             } catch (e) {
-                console.warn("Can't read the css rules of: " + sheets[i].href, e);
+                console.log("Can't read the css rules of: " + sheets[i].href, e);
```

Comment:
@qsm-odoo Ok for you? Or we remove it..
Now that we have a tour that goes through this code, the console.warn makes the test fails.

You can easily reproduce on any runbot master -> go to any page > edit mode > drag & drop text-image > double click on image.
This will raise the console warning.
```
fonts.js:39 Can't read the css rules of: https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.9.4/adyen.css DOMException: Failed to read the 'rules' property from 'CSSStyleSheet': Cannot access rules
    at Object.getCssSelectors (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web_editor.assets_wysiwyg.js:11658:35)
    at https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web_editor.assets_wysiwyg.js:11715:33
    at Function._.each._.forEach (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web.assets_common_lazy.js:151:9)
    at Object.<anonymous> (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web_editor.assets_wysiwyg.js:11714:11)
    at Object.computeFonts (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web.assets_common_lazy.js:889:21)
    at Class.init (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web_editor.assets_wysiwyg.js:15129:15)
    at Class.prototype.<computed> [as init] (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web.assets_common_lazy.js:59364:38)
    at new Class (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web.assets_common_lazy.js:59381:33)
    at Class.init (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web_editor.assets_wysiwyg.js:15662:31)
    at Class.prototype.<computed> [as init] (https://9208573-master-all.runbot32.odoo.com/web/assets/debug/1/web.assets_common_lazy.js:59364:38)
    ```

Reply:
Bof, I probably added that try...catch originally but it was not made to handle this case: like your warning shows, the code tries to read the rules of an external stuff "checkoutshopper-live" which seems to be lazy loaded by something... the code should not even try in this case :grimacing: 

So yes, at worst removing it, at best should not try to read those at all for external files..?

Reply:
I think the whole point of this try/catch is to handle external files no? https://stackoverflow.com/a/52673430

Reply:
https://github.com/odoo/odoo/commit/d434500ec8b9d5da0c341f3e6d6748a4bb7c6e2e beautiful commit message, so it was not me, strange must be confused with something else. Let's just remove the log/warn entirely IMO.

Reply:
Done